challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -996,7 +996,7 @@
         "<blockquote>const person = {<br>&nbsp;&nbsp;name: \"Zodiac Hasbro\",<br>&nbsp;&nbsp;age: 56<br>};<br><br>// Template literal with multi-line and string interpolation<br>const greeting = `Hello, my name is ${person.name}!<br>I am ${person.age} years old.`;<br><br>console.log(greeting); // prints<br>// Hello, my name is Zodiac Hasbro!<br>// I am 56 years old.<br></blockquote>",
         "A lot of things happened there.",
         "Firstly, the example uses backticks (<code>`</code>), not quotes (<code>'</code> or <code>\"</code>), to wrap the string.",
-        "Secondly, notice that the string is multi-line, both in the code and the output. This saves inserting \n within strings.",
+        "Secondly, notice that the string is multi-line, both in the code and the output. This saves inserting <code>\n</code> within strings.",
```

Comment:
Tested this locally and it seems like `\n` still needs to be escaped, whether or not it's wrapped in `<code>` tags. `<code>\\n<code>` gets it to show up locally on Learn for me.

Reply:
sorry forgot to escape the `\` will fix